### ISSUE TITLE
Delete cron line (chapter 4, p. 55) doesn't work without hour and job specified

### ISSUE THREAD

**Issue by peurKe** [geerlingguy/ansible-for-devops#183]
First, thanks for your very usefull ebook! :D 

Page 55/337
After I've added this cron line with this command in the ebook ...
> $ ansible multi -b -m cron -a "name='daily-cron-all-servers' hour=4 job='/path/to/daily-script.sh'"

... I can't delete the added cron line with this command in the ebook : 
> $ ansible multi -b -m cron -a "name='daily-cron-all-servers' state=absent"

But when I use the same first command with «state=absent» in order to delete the added cron line, it works : 
> $ ansible multi -b -m cron -a "name='daily-cron-all-servers' hour=4 job='/path/to/daily-script.sh' state=absent"

**Comment by peurKe** [comment]
I think maybe it is because there is a security mechanism in the cron module that makes it necessary to define which line should be deleted. (since the delete is based on the comment name of the cron)

**Comment by geerlingguy** [comment]
Interesting; I'll have to dig into this when I get a chance. I believe the example given did work (and I verified it did last time I touched that section of the book), so maybe it's a change in Ansible more recently.

**Comment by geerlingguy** [comment]
I just verified the commands in the book work correctly:

```
ansible -i inventory multi -b -m cron -a "name='daily-cron-all-servers' hour=4 job='/path/to/daily-script.sh'"
```

The cron job was added, and I confirmed it was present (`sudo crontab -l` on the servers). Running the command again showed no changes.

Then I ran:

```
ansible -i inventory multi -b -m cron -a "name='daily-cron-all-servers' state=absent"
```

And it reported a change on each server, and I confirmed the cron jobs were all absent (`sudo crontab -l` on the servers).

Running the `absent` command again reported no change, so it all seemed to work correctly and with idempotence.
